packaging/debian/control

```diff
@@ -118,8 +118,8 @@ Depends:
 Description: fty-rest debugging symbols
  This package contains the debugging symbols for fty-rest:
  common core rest api for 42ity project.
-
-# Customization over zproject-generated code follows:
+ 
+ # Customization over zproject-generated code follows:
```

Comment:
Leading whitespace appeared, may be toxic for debian parser. Better fix this bit back.

Reply:
Not really toxic but having a special handling as sub section / text